docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -1,6 +1,226 @@
+      integration_id = aiven_service_integration.flink_to_kafka.integration_id
+    }
+   }
+   
+   # Kafka source topic
+   
+   resource "aiven_kafka_topic" "source" {
+     project      = var.project_name
+     service_name = aiven_kafka.demo-kafka.service_name
+     partitions   = 2
+     replication  = 3
+     topic_name   = "iot_measurements"
+   }
+   
+   # Kafka sink topic
+   
+   resource "aiven_kafka_topic" "sink" {
+     project      = var.project_name
+     service_name = aiven_kafka.demo-kafka.service_name
+     partitions   = 2
+     replication  = 3
+     topic_name   = "cpu_high_usage"
+   }
+   
+
+.. dropdown:: Expand to check out how to execute the Terraform files.
+
+    The ``init`` command performs several different initialization steps in order to prepare the current working directory for use with Terraform. In our case, this command automatically finds, downloads, and installs the necessary Aiven Terraform provider plugins.
+    
+    .. code:: shell
+
+       terraform init
+
+    The ``plan`` command creates an execution plan and shows you the resources that will be created (or modified) for you. This command does not actually create any resource; this is more like a preview.
+
+    .. code:: bash
+
+       terraform plan -var-file=var-values.tfvars
+
+    If you're satisfied with the output of ``terraform plan``, go ahead and run the ``terraform apply`` command which actually does the task or creating (or modifying) your infrastructure resources. 
+
+    .. code:: bash
+
+       terraform apply -var-file=var-values.tfvars
+
+The resource ``"aiven_flink"`` creates an Aiven for Apache Flink resource with the project name, choice of cloud, an Aiven service plan, and a specified service name. 
+``"aiven_kafka"`` resource creates an Apache Kafka cluster and two Apache Kafka topics (``cpu_measurements`` and a ``cpu_high_usage``) are created using the ``"aiven_kafka_topic"`` resource.
+Similarly, the ``"aiven_service_integration"`` resource creates the integration between Apache Kafka and the Apache Flink service.
+``aiven_flink_application`` resource ``demo-flink-app`` creates the Aiven for Apache Flink application whereas ``aiven_flink_application_version`` resource ``demo-flink-app-version`` contains all the necessary specifications.
+For example, the application version resource creates two Flink tables, ``iot_measurements_table`` as the source table and ``cpu_high_usage_table`` as the sink table with the specified schema.
+
+Once the Terraform script is run, all of the resources from the manifest are created but you'll need to create the deployment. From the Aiven console, go to the ``Application`` tab under the newly created Aiven for Apache Flink service. Click on ``demo-flink-app`` and click **Create deployment**. If this is your first deployment, you won't have an option for :doc:`savepoint <../../../../products/flink/concepts/savepoints>`.
+Accept the default setting or make necessary selection and then deploy. 
```

Comment:
We might also want to say that the deployment can also be done with the Aiven CLI (https://docs.aiven.io/docs/tools/cli/service#avn-service-flink-beta)

Reply:
Hi @ftisiot I was about to make this change but then realized that the [CLI docs](https://docs.aiven.io/docs/tools/cli/service#avn-service-flink) still refer to the old API concepts like jobs.

> Manages Aiven for Apache Flink® tables and jobs.

> More info on flink table create, flink table delete, flink table get, flink table list, flink job create, flink job cancel, flink job get and flink job list can be found in

I couldn't find https://docs.aiven.io/docs/tools/cli/service#avn-service-flink-beta

Reply:
Thanks Dewan! I believe you can point people here https://docs.aiven.io/docs/tools/cli/service/flink

Reply:
Also, I have another PR to fix the list of Flink CLI commands